### PR TITLE
feat: add getInstallation command

### PR DESCRIPTION
**Pull request by bobvandenhoogen**

### CODE REVIEW

File: pkg/entity/installation/installation.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+var client installations.InstallationsServiceClient
+var apiContext context.Context
+
+func SetupClient(clientConnection *grpc.ClientConn, ctx context.Context) {
+	apiContext = ctx
+	client = installations.NewInstallationsServiceClient(clientConnection)
+}
+
+func get(_ *cobra.Command) {
+	req := &installations.GetInstallationRequest{}
```
Comment (jankeesvanandel) [inline pkg/entity/installation/installation.go:23]: I thought we added an `installation_id` parameter to GetInstallation. Are the API definitions up to date?

**Review by jankeesvanandel — CHANGES_REQUESTED** [review]

**Review by jankeesvanandel — APPROVED** [review]

**Comment by jankeesvanandel** [comment]
Looks good. I'm missing the ListInstallations operation, but maybe we don't yet need that one?

**Review by trietsch — APPROVED** [review]
LGTM

**Pull request merged**